actors/runtime/indices/indices.go

```diff
@@ -12,66 +10,22 @@ import (
 // it is a passive data structure that allows for convenience access to network indices
 // and pure functions in implementing economic policies given states
 type Indices interface {
-	Epoch() abi.ChainEpoch
-	NetworkKPI() big.Int
-	TotalNetworkSectorWeight() abi.SectorWeight
-	TotalPledgeCollateral() abi.TokenAmount
-	TotalNetworkEffectivePower() abi.StoragePower // power above minimum miner size
-	TotalNetworkPower() abi.StoragePower          // total network power irrespective of meeting minimum miner size
-
-	TotalMinedFIL() abi.TokenAmount
-	TotalUnminedFIL() abi.TokenAmount
-	TotalBurnedFIL() abi.TokenAmount
-	LastEpochReward() abi.TokenAmount
-
-	StorageDeal_DurationBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-	) (minDuration abi.ChainEpoch, maxDuration abi.ChainEpoch)
-	StorageDeal_StoragePricePerEpochBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-		endEpoch abi.ChainEpoch,
-	) (minPrice abi.TokenAmount, maxPrice abi.TokenAmount)
-	StorageDeal_ProviderCollateralBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-		endEpoch abi.ChainEpoch,
-	) (minProviderCollateral abi.TokenAmount, maxProviderCollateral abi.TokenAmount)
-	StorageDeal_ClientCollateralBounds(
-		pieceSize abi.PieceSize,
-		startEpoch abi.ChainEpoch,
-		endEpoch abi.ChainEpoch,
-	) (minClientCollateral abi.TokenAmount, maxClientCollateral abi.TokenAmount)
 	SectorWeight(
 		sectorSize abi.SectorSize,
 		startEpoch abi.ChainEpoch,
 		endEpoch abi.ChainEpoch,
 		dealWeight abi.DealWeight,
 	) abi.SectorWeight
 	PledgeCollateralReq(minerNominalPower abi.StoragePower) abi.TokenAmount
-	SectorWeightProportion(minerActiveSectorWeight abi.SectorWeight) big.Int
-	PledgeCollateralProportion(minerPledgeCollateral abi.TokenAmount) big.Int
 	StoragePower(
 		minerNominalPower abi.StoragePower,
 		minerPledgeCollateral abi.TokenAmount,
 	) abi.StoragePower
-	StoragePowerProportion(
-		minerStoragePower abi.StoragePower,
-	) big.Int
 	CurrEpochBlockReward(
 		networkIndicator big.Int,
 		networkKPI big.Int,
 		networkTime big.Int,
 	) abi.TokenAmount
-	StorageMining_PreCommitDeposit(
-		sectorSize abi.SectorSize,
-		expirationEpoch abi.ChainEpoch,
-	) abi.TokenAmount
-	NetworkTransactionFee(
-		toActorCodeID cid.Cid,
-		methodNum abi.MethodNum,
-	) abi.TokenAmount
```

Comment:
Blocking: we will still need this as a mechanism

Reply:
It doesn't happen in the actors, it happens in the interpreter.